src/frontend/src/pages/PIAForm/utils/validateForm.ts

```diff
@@ -0,0 +1,57 @@
+import { IPiaForm } from '../../../types/interfaces/pia-form.interface';
+import { PiaValidationMessage } from '..';
+import { SetStateAction } from 'react';
+
+type ValidateFormProps = {
```

Comment:
do we want to have types defined in a separate file?

Reply:
We could but this is a small file already that I think it should just stay close to its implementation